src/github/api/client.rs

```diff
@@ -42,6 +43,17 @@ impl RepositoryClient for GithubRepositoryClient {
         self.name()
     }
 
+    /// Was the comment created by the bot?
+    async fn is_comment_internal(&self, comment: &PullRequestComment) -> anyhow::Result<bool> {
+        let app = self
```

Comment:
Let's just pass `app.html_url` to the GH repository client when creating it, to avoid an unnecessary network call that could fail, and to remove `Result` from the return type. The URL should be immutable.